changelog.txt

```diff
@@ -1,3 +1,18 @@
+4.40.0
+==========
+### New Features:
+** Hybrid SPA is now production ready!! [2920]( https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2920)
+** [Hybrid SPA Sample](https://aka.ms/msal-net/hybrid-spa-sample)
+** Allow POP token envelope to be created externally [3059](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3059)
+** Remove obsolete MATS telemetry. [3043](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3043)
+** Windows Forms are referenced only in Desktop [3071](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/commit/c240dad04ead8cb7a729ffcb5d23f0b02bf582b5)
+** Log clarification in several places.
```

Comment:
What is this in reference to? Add a link to related issue or PR.

Reply:
There are two places. Added refs